graphics/timg/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup cmake 1.1
+PortGroup github 1.0
+
+revision 1
+
+categories graphics
+maintainers @urkle
+license GPL-2
+
+description Terminal Image and Video Viewer
+
+long_description A user-friendly terminal image viewer that uses graphic \
+                 capabilities of terminals (Sixel, Kitty or iterm2), or 24-Bit \
+                 color capabilities and unicode character blocks if these are not \
+                 available.
+
+homepage https://timg.sh/
+
+github.setup hzeller timg 1.6.0 v
+github.tarball_from archive
+
+checksums           rmd160  fc1b39f4395f7a4dc4da77f12e558289e05f3425 \
+                    sha256  9e1b99b4eaed82297ad2ebbde02e3781775e3bba6d3e298d7598be5f4e1c49af \
+                    size    1166246
+
+depends_build-append path:bin/pkg-config:pkgconfig
+
+depends_lib-append port:cmake \
+                   port:git \
```

Comment:
I haven't checked the dependencies, but why are "cmake" and "git" library dependencies?
CMake is typically a build dependency and already added since you included the cmake PortGroup. 

Additionally, severalmof these dependencies has a "-devel" port and you want to make sure these can satisfy the dependency as well. Check other ports that have the same dependencies on the preferred/exact syntax.